gemhardware/src/common/glib/GLIBManager.cc

```diff
@@ -162,6 +162,7 @@ void gem::hw::glib::GLIBManager::initializeAction()
   throw (gem::hw::glib::exception::Exception)
 {
   DEBUG("GLIBManager::initializeAction begin");
+  // FIXME make me more streamlined
```

Comment:
I suggest to use range-based loop here and later for alike places

Reply:
It's not quite as simple as that, as I may have mentioned before.
The `SOAP` commands and the `parameterGet`/`parameterSet` commands are expecting the populated object at specific indices in the `SOAP` message
So as has been my hope/plan for a while, this will require a bit of careful thought to make sure it gets handled properly.